x-pack/plugins/apm/server/lib/traces/get_trace_items.ts

```diff
@@ -88,7 +88,7 @@ export async function getTraceItems(
     // explicit intermediary types to avoid TS "excessively deep" error
     PromiseValueType<typeof errorResponsePromise>,
     PromiseValueType<typeof traceResponsePromise>
-  ] = await Promise.all([errorResponsePromise, traceResponsePromise]);
+  ] = (await Promise.all([errorResponsePromise, traceResponsePromise])) as any;
```

Comment:
This was randomly failing on my machine when building the entire project, so I've added a `as any` as we are explicitly typing it anyway.